engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/service/PeriodicProcessListener.scala

```diff
@@ -25,11 +25,11 @@ sealed trait PeriodicProcessEvent {
 
 case class DeployedEvent(deployment: PeriodicProcessDeployment, externalDeploymentId: Option[ExternalDeploymentId]) extends PeriodicProcessEvent
 
-case class FinishedEvent(deployment: PeriodicProcessDeployment, processState: Option[ProcessState]) extends PeriodicProcessEvent
+case class FinishedEvent(deployment: PeriodicProcessDeployment, processState: Option[StatusDetails]) extends PeriodicProcessEvent
```

Comment:
We have to mention about this in migration guide

Reply:
Done.